modules/explorer/info.go

```diff
@@ -38,6 +38,26 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() (modules.BlockFacts, bool) {
```

Comment:
Yeah, @lukechampine and I had a brief offline conversation about this.  Would returning an error be saner?